pagination/src/main/java/com/cerner/beadledom/pagination/models/OffsetPaginationConfiguration.java

```diff
@@ -37,6 +37,11 @@
    */
   public abstract String offsetFieldName();
 
+  /**
+   * Flag to denote whether limit value of 0 is allowed. Defaults to false.
+   */
+  public abstract boolean allowZeroLimit();
```

Comment:
@venki09 I switched this from `Boolean` to `boolean` since it can't be nullable, but I wanted to double check that there wasn't a reason I'm missing for it being a big boolean?

Reply:
No there was no reason for it. I think it makes sense to have it boolean